cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -524,6 +525,17 @@ pGenesisCmd =
             <> Opt.help "The genesis start time in YYYY-MM-DDThh:mm:ssZ format. If unspecified, will be the current time +30 seconds."
             )
 
+    pGenesisDelegates :: Parser Word
+    pGenesisDelegates =
+      fromMaybe 7 <$> Opt.optional
+        ( Prelude.read <$>
+            Opt.strOption
+              (  Opt.long "genesis-delegates"
```

Comment:
```suggestion
              (  Opt.long "gen-genesis-delegate-keys"
```
Perhaps, to better hint that it actually creates these new keys, rather than using existing supplied ones.